client/src/main/java/io/pravega/client/control/impl/ModelHelper.java

```diff
@@ -464,12 +464,12 @@ public static final NodeUri decode(final PravegaNodeUri uri) {
         Preconditions.checkNotNull(groupName, "ReaderGroup name is null");
         Preconditions.checkNotNull(config, "ReaderGroupConfig is null");
 
-        List<StreamCut> startStreamCuts = config.getStartingStreamCuts().entrySet().stream()
+        List<Controller.StreamCut> startStreamCuts = config.getStartingStreamCuts().entrySet().stream()
```

Comment:
nit: revert this

Reply:
This is necessary because this class deals with 2 kind of StreamCut class objects - one from Controller and another one from Client.